### DISCUSSION TITLE
How to move a Scene to the Featured Scenes in Hubs Cloud??

### DISCUSSION THREAD

**Discussion by BonzoMartini** [mozilla/hubs#3282] [Hubs Cloud Questions] [answered]
So just that - how to move a 'Scene' to the 'Featured Scenes' in Hubs Cloud?? & How to set which scene the user arrives at when they hit the domain. I've read the docs, it's just confusing and a diagram would be a big help

**Comment by robinkwilson** [comment, marked as answer]
Click the "Feature" button in **Approved Scenes** menu.

![image](https://user-images.githubusercontent.com/29560735/98304797-8486e600-1f75-11eb-8e53-1c67a00d1122.png)

Likewise, if you want to unfeature a scene, click the "Unfeature" in the **Featured Scenes** menu.

**Comment by BonzoMartini** [comment]
Thankyou. I must comment this is a very ambiguous interface - I did not realise that FEATURE was a button, I thought it was an info message like  'delisted ' whereas EDIT really looks like it does something. So in my Approved Scenes list, scenes which are actually  _Featured_ are marked _unfeature_ Lol. 
Suggestion would not something like the video type _Play / Stop_ be more obvious or move the featuring function into"EDIT"

**Reply by robinkwilson** [reply]
Totally agree! If you'd like to make a feature request issue in [hubs-cloud repo](https://github.com/mozilla/hubs-cloud), you're welcome to! There's honestly a lot that we'd like to change in the Admin Panel, just haven't had the time!
